### ISSUE TITLE
Implement Flexible SVG Icon Component

### ISSUE THREAD

**Issue by axzilla** [axzilla/goilerplate#1]
# Flexible SVG Icon Component

## Description
Create a versatile SVG Icon component that can support various icon libraries and custom SVGs, enhancing icon usage consistency across the project.

## Key Features
- Universal component for rendering SVG icons
- Support for multiple icon libraries (e.g., Lucide, Heroicons, custom SVGs)
- Customizable size, color, and additional CSS classes

## Rough Implementation Idea
```go
templ Icon(props IconProps) {
    <svg 
        xmlns="http://www.w3.org/2000/svg" 
        width={props.Size} 
        height={props.Size} 
        viewBox="0 0 24 24" 
        class={props.Class}
    >
        // Logic to render icon based on chosen library or custom path
    </svg>
}

**Comment by axzilla** [comment]
This issue has been implemented in release x.y.z. See [https://github.com/axzilla/goilerplate/releases/tag/v0.20.0].

**Comment by axzilla** [comment]
This issue has been implemented in release v0.20.0. See https://github.com/axzilla/goilerplate/releases/tag/v0.20.
